package.json

```diff
@@ -22,13 +22,28 @@
     "type": "git",
     "url": "https://github.com/cloudfour/cloudfour.com-patterns.git"
   },
+  "main": "./dist/cloudfourPatterns.js",
+  "unpkg": "./dist/cloudfourPatterns.min.js",
+  "module": "./dist/cloudfourPatterns.mjs",
+  "exports": "./dist/cloudfourPatterns.mjs",
+  "files": [
+    "/dist",
+    "/src/**/*.scss",
+    "/src/**/*.twig",
+    "!/src/**/demo/**/*.twig",
+    "/src/assets",
+    "!.gitignore",
+    "!src/index.scss"
+  ],
   "dependencies": {
     "focus-visible": "5.0.2",
     "prismjs": "1.19.0"
   },
   "devDependencies": {
     "@babel/core": "7.9.0",
     "@cloudfour/eslint-plugin": "8.0.0",
+    "@rollup/plugin-multi-entry": "3.0.0",
```

Comment:
Are we using rollup and gulp? That seems odd.

Reply:
Yes, we are using both, they serve different purposes.

We are using Gulp as a task runner, to pipe code from one tool to another.
We are using Rollup as a bundler, to resolve dependencies of js code, tree-shake, and combine.

Reply:
@spaceninja, I'm curious, have you used Rollup without a task runner? In the past I've done this with Webpack, relying on NPM scripts for orchestration. But @calebeby set up other projects with this combination, and it seems like a good approach.

Reply:
I have very little experience, I was asking because in my head I think of Rollup and Gulp as basically doing the same thing. But this approach makes sense (and I trust Caleb)

Reply:
When I started doing this kind of stuff (not nearly as complex as what @calebeby is doing), I was using Grunt and then brought Webpack into that system. After I while, I wanted to ditch Grunt and turned to NPM scripts (and some script utilities) to orchestrate things, but I think Gulp provides a cleaner, more composable way of doing this.